### ISSUE TITLE
Problem playing 180° VR video

### ISSUE THREAD

**Issue by Svenbendt2** [Igalia/wolvic#575]
## Configuration

Wolvic version: 1.1
Wolvic build ID: 48

Hardware: VIVE Flow

## Steps to Reproduce

1. Open URL https://virtualrealpassion.com/vr-female-pov-porn-video/improvised-striptease/
2. Play video

## Current Behavior

Video is trying to play ... sound is playing ... no video

## Expected Behavior

Video and sound playing

## Error Logs and Stack Traces

<details open>

```
```

</details>

**Comment by svillar** [comment]
First of all thanks for reporting, this makes wolvic better.

The site you mention is using the DelightXR player for playing the video. We indeed had issues with that player but they were fixed (see #370) in release 1.3. You're using 1.1 which is rather old. So if you manage to get a 1.3 release this issue will be fixed for you (note that Vive is not officially supported so I am not sure where you are getting the packages from).
